pkg/controller/migmigration/task.go

```diff
@@ -1,29 +1,44 @@
 package migmigration
 
 import (
+	"context"
 	"fmt"
 	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
 	"github.com/go-logr/logr"
 	velero "github.com/heptio/velero/pkg/apis/velero/v1"
+	corev1 "k8s.io/api/core/v1"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // Annotation Keys
-const MigQuiesceAnnotationKey = "openshift.io/migrate-quiesce-pods"
+const (
+	MigQuiesceAnnotationKey        = "openshift.io/migrate-quiesce-pods"
+	copyBackupRestoreAnnotationKey = "openshift.io/copy-backup-restore"
+)
 
 // Phases
 const (
 	Started                 = "Started"
 	WaitOnResticRestart     = "WaitOnResticRestart"
 	ResticRestartCompleted  = "ResticRestartCompleted"
-	BackupStarted           = "BackupStarted"
-	BackupCompleted         = "BackupCompleted"
-	BackupFailed            = "BackupFailed"
+	InitialBackupStarted    = "InitialBackupStarted"
+	InitialBackupCompleted  = "InitialBackupCompleted"
+	InitialBackupFailed     = "InitialBackupFailed"
+	CopyBackupStarted       = "CopyBackupStarted"
+	CopyBackupCompleted     = "CopyBackupCompleted"
+	CopyBackupFailed        = "CopyBackupFailed"
+	CreatingStagePods       = "CreatingStagePods"
```

Comment:
`CreateStagePods` would be more consistent with other phases.

Reply:
Or possibly `CreateStagePodsStarted` and `CreateStagePodsCompleted` for this one and the one following.